assets/src/stories-editor/components/higher-order/with-right-click-handler.js

```diff
@@ -36,17 +36,27 @@ const applyWithDispatch = withDispatch( ( dispatch, ownProps, { select } ) => {
 			return;
 		}
 
+		const editLayout = document.querySelector( '.edit-post-layout' );
 		if ( ! document.getElementById( 'amp-story-right-click-menu' ) ) {
-			const editLayout = document.querySelector( '.edit-post-layout' );
-
 			const menuWrapper = document.createElement( 'div' );
 			menuWrapper.id = 'amp-story-right-click-menu';
 
 			editLayout.appendChild( menuWrapper );
 		}
 
+		// Calculate the position to display the right click menu.
```

Comment:
If it helps, the `Popover` component has utils to do similar position calculation:

https://github.com/WordPress/gutenberg/blob/02c5f3950dda05ffbba532d285e58be9232c0acf/packages/components/src/popover/utils.js

Reply:
Thanks, will check again tomorrow.